ereadingtool/ereadingtool
# Corrupted font files result in FF error

**What it does**
Accidentally had an extra directory filled with WOFF2 files. Deleted the WOFF2 files that were incompatible with Firefox. Grabbed OpenSans file from FontSquirrel, it's WOFF not WOFF2 but it should work.

**How to test it**
Can't seem to reproduce locally because the font is installed on my machine. However, if this problem shows for you there will be an error in the browser's console. I do see the error when the files are delivered from a server.

If unable to reproduce, we could use the development server and pull from this branch.

**Which files were affected**
A = add D = deleted
```
D       web/public/fonts/fonts/eb-garamond-bold.woff2
D       web/public/fonts/fonts/eb-garamond-italic.woff2
D       web/public/fonts/fonts/eb-garamond-regular.woff2
D       web/public/fonts/fonts/opensans-bold.woff2
D       web/public/fonts/fonts/opensans-italic.woff2
D       web/public/fonts/fonts/opensans-regular.woff2
A       web/public/fonts/opensans-bold.woff
D       web/public/fonts/opensans-bold.woff2
A       web/public/fonts/opensans-italic.woff
D       web/public/fonts/opensans-italic.woff2
A       web/public/fonts/opensans-regular.woff
D       web/public/fonts/opensans-regular.woff2
```
